### PR TITLE
add Privileged test case

### PR DESCRIPTION
**Pull request by Helen-Xie**
part of #20 

Signed-off-by: Helen-Xie <yulinxie@zju.edu.cn>

### CODE REVIEW

**Comment by Helen-Xie** [comment]
ping @feiskyer

File: pkg/validate/security_context.go [hunk 1]
```diff
@@ -279,6 +279,42 @@ var _ = framework.KubeDescribe("Security Context", func() {
 			By("Check whether rootfs is read-only")
 			checkRootfs(podConfig, logPath, readOnlyRootfs)
 		})
+
+		It("runtime should support Privileged is true", func() {
+			By("create pod")
+			podID, podConfig = createPodSandboxForContainer(rc)
```
Comment (feiskyer) [inline pkg/validate/security_context.go]: Sandbox should also privileged for this case.

File: pkg/validate/security_context.go [hunk 2]
```diff
@@ -428,3 +464,38 @@ func compareNetworkList(podConfig *runtimeapi.PodSandboxConfig, logPath string,
...
+	reverseCmd := []string{"ip", "link", "del", "dummy0"}
+
+	stdout, stderr, err := rc.ExecSync(containerID, cmd, time.Duration(defaultExecSyncTimeout)*time.Second)
+	msg := fmt.Sprintf("cmd %v, stdout %q, stderr %q", cmd, stdout, stderr)
+
+	if isPrivileged {
+		Expect(err).NotTo(HaveOccurred(), msg)
+
+		// We need to clean up the dummy link that was created.
+		execSyncContainer(rc, containerID, reverseCmd)
```
Comment (feiskyer) [inline pkg/validate/security_context.go]: I think this is not required since we will remove the container/sandbox after each test case.

Comment (Helen-Xie) [inline pkg/validate/security_context.go]: ok, I will update it.

**Comment by feiskyer** [comment]
ReadOnlyRootfs and HostNetwork test failed randomly.

**Comment by Helen-Xie** [comment]
update. ping @feiskyer 

the command execution time is too short, so func `Eventually()` of the containerStatus of the ReadOnlyRootfs and HostNetwork test case will failed randomly. I think there is not required `Eventually()`, because there is no need to keep the container always `running`.

**Comment by xlgao-zju** [comment]
@feiskyer wired, exec test failed, and get passed when I restart it...
I think there is nothing wrong with exec test, the failure is [line](https://github.com/kubernetes-incubator/cri-tools/blob/master/pkg/validate/streaming.go#L148).

**Comment by feiskyer** [comment]
LGTM

**Pull request merged**